src/utils/metadata.ts

```diff
@@ -99,6 +99,22 @@ function getAlgorithmFileExtension(fileUrl: string): string {
   return splitedFileUrl[splitedFileUrl.length - 1]
 }
 
+function sanitizeUrl(url: string): string {
+  return url.replace('javascript:', '')
```

Comment:
question(blocking): Does the xss attack still work when the `'javascript:'` is written several times? 
```
javascript:javascript:
```

If pattern in `replace()` is a string, only the first occurrence will be replaced.